install

```diff
@@ -10,10 +10,13 @@ function install_env {
     ENV=$1
     tk env add environments/$ENV --server-from-context $CONTEXT --namespace $ENV # Creates/configures the environment.
     ln -sf "$PROD/$ENV/main.jsonnet" environments/$ENV/main.jsonnet # Copies the base configuration for your env.
+		if [[ $# > 1 ]]; then
```

Comment:
what does this check do?

Reply:
check if we have an extra parameter to fill 'config.jsonnet'.